gradle/libs.versions.toml

```diff
@@ -241,7 +241,7 @@ auth = "com.google.android.gms:play-services-auth:20.4.0"
 barista = "com.adevinta.android:barista:4.2.0"
 browser = "androidx.browser:browser:1.4.0"
 browser-helper = "com.google.androidbrowserhelper:androidbrowserhelper:2.3.0"
-car = "androidx.car.app:app:1.2.0"
+car = "androidx.car.app:app:1.4.0-rc02"
```

Comment:
Updated to fix this error on creating a bookmark:

<img width="1229" alt="Screenshot 2024-05-29 at 3 40 25 PM" src="https://github.com/Automattic/pocket-casts-android/assets/1405144/539eaf68-8916-46f2-8f28-9bbf841214b5">